py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -692,7 +692,9 @@ def aten_ops_softmax(
 
 @dynamo_tensorrt_converter(
     torch.ops.aten.split.Tensor,
-    capability_validator=has_static_shapes_in_args([1]),
+    capability_validator=(
+        has_static_shapes_in_args([0]) and has_static_shapes_in_args([1])
```

Comment:
args[0] is `input` and args[1] is `split_size_or_sections`. Initially I thought we should add this since dynamic input tensor split is not supported, but yes you are right, the error comes in torch.export before the interpretor stage, so having this validator does not make sense. We are anyways skipping the test so that CI does not fail. So will remove this.